pmpsui/main.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import logging
+from pathlib import Path
+
+from pydm import PyDMApplication
+from pydm.utilities.macro import parse_macro_string
+
+logger = logging.getLogger(__name__)
+
+def make_parser():
+    parser = argparse.ArgumentParser(
+        description='Display the PMPS diagnostic tool inside of PyDM.',
+        prog='pydm pmps.py',
+    )
+    parser.add_argument(
+        '--no-web',
+        action='store_true',
+        help='Disable the grafana web view tab.',
+    )
+
+    parser.add_argument(
+        '--macro',
+        help=("Macro subsitution to use, in JSON object format.  Same as PyDM"
+              "macro substitution")
+    )
```

Comment:
I think it might be time (soon? in another PR?) to move past this `--macro` raw interface that forces me to e.g. `--macro CFG=KFE` and let the user here simply pick which config to use.